storage/v1alpha1/register.go

```diff
@@ -0,0 +1,46 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+const (
+	Version   = "v1alpha1"
+	GroupName = "storage.openshift.io"
```

Comment:
@jsafrane can you ACK claiming this name?

Reply:
I don't think that the `SharedResource` belongs to `storage.openshift.io`. It's used to authorize usage of a Secret / ConfigMap in another namespace and has very little to do with actual storage (despite it's referenced in a `volumes` section of a Pod). `share.openshift.io`?

Reply:
We had a round of iteration in that same path previously: https://github.com/openshift/api/pull/979#issuecomment-889326398

Reply:
And this is how we arrived at `storage.openshift.io`: https://github.com/openshift/api/pull/979#issuecomment-906633168

Reply:
I see. You can see, naming is hard. CSI (and storage in general) is just an implementation detail of SharedResource. In theory you could do it without CSI too with copying Secrets around and having an (ugly) admission webhook. Intent of the API is not to provide storage, it provides shared secrets / config maps.